packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -294,7 +294,16 @@ class OnsetPicker extends React.Component {
     const { intl } = this.context;
 
     let granularitySelect = null;
+    const birthMoment = moment(birthdate).startOf('day'); // startOf to clear time from values
+    const currentMoment = moment().startOf('day');
     if (this.state.precision !== PrecisionOptions.UNKNOWN) {
+      const optionAge = (currentMoment.diff(birthMoment, 'weeks') !== 0) ? (
```

Comment:
Since condition for age less than a week is being handled here now, do we still need to the same check in [allowedAgeUnits](https://github.com/cerner/terra-clinical/blob/8156586a2a2eb261d4f732c19f76a5ec0b52d39c/packages/terra-clinical-onset-picker/src/OnsetUtils.js#L92) method of OnsetUtils.js

Reply:
Fixed here https://github.com/cerner/terra-clinical/pull/543/commits/dea7f66925ff7547256295d24794eb437c9dc8ca.